js/index.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+
+  $('a').click(function(){
```

Comment:
The reason this isn't working is because in order for an anchor link to work it needs an ID to point to, not a class. If you change line 36 of your index.html to be `id="content"` instead of `class="content"` the smooth scrolling will work.